bindings_ffi/src/mls.rs

```diff
@@ -8057,8 +8058,8 @@ mod tests {
             .await
             .unwrap();
 
-        assert_eq!(final_bo_messages.len(), 5, "Bo should see 5 messages");
-        assert_eq!(final_alix_messages.len(), 5, "Alix should see 5 messages");
+        assert_eq!(final_bo_messages.len(), 6, "Bo should see 5 messages");
+        assert_eq!(final_alix_messages.len(), 6, "Alix should see 5 messages");
```

Comment:
Same here I think this should still be 5. We should not stitch the group update message.